src/offchainapi/tests/test_payment_logic.py

```diff
@@ -161,7 +246,7 @@ def test_payment_process_abort_from_receiver(payment, processor):
     bcm = processor.business_context()
     bcm.is_recipient.side_effect = [True, True]
     bcm.check_account_existence.side_effect = [None]
-    bcm.next_kyc_level_to_request.side_effect = [BusinessForceAbort]
+    bcm.next_kyc_level_to_request.side_effect = [ BusinessForceAbort(OffChainErrorCode.payment_insufficient_funds, 'Not enough money in account.') ]
```

Comment:
this is trivial but `payment_insufficient_funds` usually is not thrown in `next_kyc_level_to_request` :)

Reply:
Indeed -- but its a good enough place to bubble this exception up the payment logic :)